doc/validation.rst

```diff
@@ -22,44 +24,55 @@ command line options for this hook:
 
     $ python -m numpydoc.hooks.validate_docstrings --help
 
-Using a config file provides additional customization. Both
-``pyproject.toml`` and ``setup.cfg`` are supported; however, if the
-project contains both you must use the ``pyproject.toml`` file.
-The example below configures the pre-commit hook to ignore three checks
-and specifies exceptions to the checks ``SS05`` (allow docstrings to
-start with "Process ", "Assess ", or "Access ") and ``GL08`` (allow
-the class/method/function with name "__init__" to not have a docstring).
+Using a config file provides additional customization. Both ``pyproject.toml``
+and ``setup.cfg`` are supported; however, if the project contains both
+you must use the ``pyproject.toml`` file. The example below configures
+the pre-commit hook as follows:
+
+* ``checks``: Report findings on all checks except ``EX01``, ``SA01``, and
+  ``ES01`` (using the same logic as the :ref:`validation during Sphinx build
+  <validation_during_sphinx_build>` for ``numpydoc_validation_checks``).
+* ``exclude``: Don't report issues on objects matching any of the regular
+  regular expressions ``\.undocumented_method$`` or ``\.__repr__$``. This
+  maps to ``numpydoc_validation_exclude`` from the
+  :ref:`Sphinx build configuration <validation_during_sphinx_build>`.
+* ``override_SS05``: Allow docstrings to start with "Process ", "Assess ",
+  or "Access ". To override different checks, add a field for each code in
+  the form of ``override_<code>`` with a collection of regular expression(s)
+  to search for in the contents of a docstring, not the object name. This
+  maps to ``numpydoc_validation_overrides`` from the
+  :ref:`Sphinx build configuration <validation_during_sphinx_build>`.
 
 ``pyproject.toml``::
 
     [tool.numpydoc_validation]
-    ignore = [
+    checks = [
```

Comment:
I've always found the numpydoc convention confusing; I think the exclusion list you had before is more logical.